client/signup/processing-screen/index.jsx

```diff
@@ -231,7 +235,7 @@ export class SignupProcessingScreen extends Component {
 	}
 
 	render() {
-		if ( abtest( 'postSignupUpgradeScreen' ) === 'modified' && ! this.state.hasPaidSubscription && this.props.flow !== 'rebrand-cities' ) {
+		if ( abtest( 'postSignupUpgradeScreen' ) === 'modified' && ! this.state.hasPaidSubscription && this.currentFlowIncludesDomainStep() ) {
```

Comment:
Is that change in logic intentional?

Reply:
@akirk Yes, it was my intention. Like _rebrand-cities_, flows that don't create a custom domain should skip the new post-signup screen.

Reply:
Ok, I just think for clarity in the PR it would have been useful to explain that `this.props.flowSteps.indexOf( 'domains' ) !== -1` covers the `this.props.flow !== 'rebrand-cities'` case also.